js/fetch_test.ts

```diff
@@ -43,3 +43,142 @@ testPerm({ net: true }, async function fetchBlob() {
   assertEqual(blob.type, headers.get("Content-Type"));
   assertEqual(blob.size, Number(headers.get("Content-Length")));
 });
+
+// Logic heavily copied from web-platform-tests, make 
+// sure pass mostly header basic test
+/* tslint:disable-next-line:max-line-length */
+// ref: https://github.com/web-platform-tests/wpt/blob/7c50c216081d6ea3c9afe553ee7b64534020a1b2/fetch/api/headers/headers-basic.html
+// tslint:disable:only-arrow-functions
+test(function() {
+  try {
+    const h = new Headers();
+  } catch (e) {
+    assert(false, "Create headers from no parameter failed");
+  }
```

Comment:
You don't need try catch blocks for successful tests. The `test()` function already does that.

You do need to add a name to the function, because that is used in the test output. So, for example, this test should be something like:
```js
test(function newHeadersSuccess() {
    new Headers();
});
```